### PR TITLE
Added window resizing redraws to curses mode

### PR DESCRIPTION
**Pull request by ramo-j**
Added a handler for signal.SIGWINCH (terminal resizing) that redraws the screen.

### CODE REVIEW

File: dftimewolf/cli/curses_display_manager.py [hunk 1]
```diff
@@ -210,23 +213,36 @@ def EnqueueMessage(self,
...
-                             subsequent_indent='  ',
-                             replace_whitespace=False)
-
-    for line in lines:
       if line:
         self._messages.append(Message(source, line, is_error))
 
     self.Draw()
 
+  def PrepareMessagesForDisplay(self, space: int) -> List[str]:
```
Comment (rgayon) [inline dftimewolf/cli/curses_display_manager.py:222]: Nit: can you change "space" to something more explicit?
maybe something like "tty_max_lines" ?

**Review by rgayon — APPROVED** [review]
LGTM, Just a nit

File: dftimewolf/cli/curses_display_manager.py (hunk 1 above)
Comment (ramo-j) [reply dftimewolf/cli/curses_display_manager.py]: Done.

**Comment by ramo-j** [comment]
@rgayon can you please re-review, your approval was discarded when adding the requested change.

**Review by rgayon — APPROVED** [review]
LGTM (and sorry for the delay)

**Pull request merged**